test/ChainlinkPriceOracle.t.sol

```diff
@@ -4,17 +4,17 @@ pragma solidity 0.8.19;
 import { Test } from "forge-std/Test.sol";
 import { AggregatorV3Interface } from "@smartcontractkit/chainlink/interfaces/AggregatorV3Interface.sol";
 import { IPriceOracle } from "../contracts/Oracles/Interfaces/IPriceOracle.sol";
-import { ChainlinkPriceOracleWETH } from "../contracts/Oracles/ChainlinkPriceOracleWETH.sol";
+import { ChainlinkPriceOracle } from "../contracts/Oracles/ChainlinkPriceOracle.sol";
 
-contract ChainlinkPriceOracleWETHTest is Test {
+contract ChainlinkPriceOracleTest is Test {
```

Comment:
I wouldn't rename this, how and for other coins, we could add tests to see is setup good.